### PR TITLE
Adding missing quotes to setup-git-sdk.bat

### PR DESCRIPTION
**Pull request by evandrocoan**
https://github.com/evandroforks/anki/runs/524380395?check_suite_focus=true#step:2:114

### CODE REVIEW

File: sdk-installer/setup-git-sdk.bat [hunk 1]
```diff
@@ -5,13 +5,13 @@
 @REM https://technet.microsoft.com/en-us/library/bb490909.aspx says:
 @REM <percent>~dpI Expands <percent>I to a drive letter and path only.
 @REM <percent>~fI Expands <percent>I to a fully qualified path name.
-@FOR /F "delims=" %%D in ("%~dp0") do @set cwd=%%~fD
+@FOR /F "delims=" %%D in ("%~dp0") do @set "cwd=%%~fD"
```
Comment (dscho) [inline sdk-installer/setup-git-sdk.bat:8]: Sorry for missing this PR!

So I guess the underlying issue is that the script does not run correctly e.g. when installing into a path whose name has spaces in it?

I am not sure if both this change and the quoting of the `PATH` assignment are needed... When I run

```cmd
set PATH=%PATH%;C:\with spaces
```

it works over here...

File: sdk-installer/setup-git-sdk.bat [hunk 2]
```diff
@@ -31,8 +31,8 @@
 
 @REM Avoid overlapping address ranges
 @IF 32 == @@BITNESS@@ @(
-	ECHO Auto-rebasing .dll files
-	CALL autorebase.bat
+    ECHO Auto-rebasing .dll files
+    CALL autorebase.bat
```
Comment (dscho) [inline sdk-installer/setup-git-sdk.bat:35]: Please drop this hunk, it is only changing white-space and that is orthogonal to the purpose of this PR.

File: sdk-installer/setup-git-sdk.bat [hunk 3]
```diff
@@ -47,4 +47,3 @@
 @ECHO Installation of Git for Windows' SDK failed!
 @PAUSE
 @EXIT /B 1
-
```
Comment (dscho) [inline sdk-installer/setup-git-sdk.bat:50]: Please drop this white-space only hunk, too.

File: sdk-installer/setup-git-sdk.bat (hunk 2 above)
Comment (evandrocoan) [reply sdk-installer/setup-git-sdk.bat:35]:
```suggestion
	CALL autorebase.bat
```

Comment (evandrocoan) [inline sdk-installer/setup-git-sdk.bat:35]:
```suggestion
	CALL autorebase.bat
```

File: sdk-installer/setup-git-sdk.bat [hunk 4]
```diff
@@ -31,8 +31,8 @@
 
 @REM Avoid overlapping address ranges
 @IF 32 == @@BITNESS@@ @(
-	ECHO Auto-rebasing .dll files
-	CALL autorebase.bat
+    ECHO Auto-rebasing .dll files
```
Comment (evandrocoan) [inline sdk-installer/setup-git-sdk.bat:34]:
```suggestion
	ECHO Auto-rebasing .dll files
```

File: sdk-installer/setup-git-sdk.bat [hunk 5]
```diff
@@ -47,4 +47,3 @@
 @ECHO Installation of Git for Windows' SDK failed!
 @PAUSE
 @EXIT /B 1
```
Comment (evandrocoan) [inline sdk-installer/setup-git-sdk.bat:49]:
```suggestion
@EXIT /B 1

```

File: sdk-installer/setup-git-sdk.bat (hunk 1 above)
Comment (evandrocoan) [reply sdk-installer/setup-git-sdk.bat:8]: Running `et PATH=%PATH%;C:\with spaces` will set the path to `et PATH=%PATH%;C:\with`

**Comment by evandrocoan** [comment]
What is `<copy-from-sign-off>`?

**Comment by PhilipOakley** [comment]
`<copy-from-sign-off>` is an instruction that you should use, in that place, your formal name and email address, as used in the DCO 'sign-off', so as to make you the formal author as a proper record of your contribution. Without that, other readers will see at as a random drive-by change. Check the DCO for what it means.

**Comment by dscho** [comment]
> `<copy-from-sign-off>` is an instruction that you should use, in that place, your formal name and email address, as used in the DCO 'sign-off', so as to make you the formal author as a proper record of your contribution. Without that, other readers will see at as a random drive-by change. Check the DCO for what it means.

Right. But I overlooked something really important: the Signed-off-by: line is expected to contain a valid email address. So the right thing here would be to call

```sh
git commit --amend --author="Evandro Coan <evandrocoan@hotmail.com>" -s
```

and delete the "Signed-off-by:" line with the URL in it.

While at it, the second commit should be squashed into the first. There is no need to keep the chronological history of the patch in the commit history. In other words:

```sh
git reset --soft HEAD^
git commit --amend --author="Evandro Coan <evandrocoan@hotmail.com>" -s
```

**Comment by evandrocoan** [comment]
I won't put my email in plain text anywhere in a commit history. I added:
```
Signed-off-by: Evandro Coan <https://github.com/evandrocoan>
```

If you want to know my email just access my profile `https://github.com/evandrocoan` and you will find it.

**Comment by evandrocoan** [comment]
@dscho, Please remove my email from your message. I already receive enough spam for several lifetimes.

**Comment by dscho** [comment]
> Please remove my email from your message.

Done.

> I already receive enough spam for several lifetimes.

Yeah, it is really annoying. I totally understand.
